packages/pharos/src/components/dropdown-menu-nav/pharos-dropdown-menu-nav.ts

```diff
@@ -19,11 +19,11 @@ import FocusMixin from '../../utils/mixins/focus';
  */
 export class PharosDropdownMenuNav extends FocusMixin(PharosElement) {
   /**
-   * Indicates the aria label to apply to the nav.
-   * @attr label
+   * Indicates the aria label to apply to the button.
```

Comment:
Will a consumer using PharosDropdownMenuNav know what `button` is being referred to here? I suspect not, personally. Might do with a bit more elaboration.

Reply:
Oof, you are correct, a lil bit of copy/pasta action happening here. Will update accordingly!